release/nightly_tests/setup_chaos.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+import ray
+
+from ray._private.test_utils import get_and_run_node_killer
+
+
+def parse_script_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--node-kill-interval", type=int, default=60)
+    parser.add_argument(
+        "--no-start",
+        action="store_true",
+        default=False,
+        help=("If set, node killer won't be starting to kill nodes when "
+              "the script is done. Driver needs to manually "
+              "obtain the node killer handle and invoke run method to "
+              "start killing nodes. If not set, as soon as "
+              "the script is done, nodes will be killed every "
+              "--node-kill-interval seconds."))
+    return parser.parse_known_args()
+
+
+def main():
```

Comment:
Is there a way we could report how many nodes were killed at the end of the test?

Reply:
This is difficult without modifying all tests since we need to get detached actor to get the information.

But I think we can probably use some "hack" to achieve it. Let me try if I can get that working.

Reply:
This is not so easy unfortunately. I think eventually, we'd like to make some hook on each release test to allow this sort information to be printed (e.g., peak memory usage or # of nodes killed). For now, I will manually print this at the end of the chaos test (instead of supporting as a fixture level)

Reply:
Added manually for now. Will make a way to allow better fixture in the sooner future.